hpy/debug/src/autogen_debug_ctx_init.h

```diff
@@ -157,8 +157,8 @@ void debug_ctx_Tracker_ForgetAll(HPyContext *dctx, HPyTracker ht);
 void debug_ctx_Tracker_Close(HPyContext *dctx, HPyTracker ht);
 void debug_ctx_Field_Store(HPyContext *dctx, DHPy target_object, HPyField *target_field, DHPy h);
 DHPy debug_ctx_Field_Load(HPyContext *dctx, DHPy source_object, HPyField source_field);
-HPyThreadState debug_ctx_LeavePythonExecution(HPyContext *dctx);
 void debug_ctx_ReenterPythonExecution(HPyContext *dctx, HPyThreadState state);
+HPyThreadState debug_ctx_LeavePythonExecution(HPyContext *dctx);
```

Comment:
Ok, so it won't be that simple, because we have a CI job that checks that someone did not forget to `make autogen` before pushing, which is think is a good thing in general. I am not so sure about alphabetical or some other artificial order. The order as it is has some logic to it, maybe we want to preserve it even in generated header, which some people may still want to inspect manually..

Reply:
But if I read the diff correctly, you didn't change the ordering inside `public_api.h`, did you?
So, it means that the order inside the autogen files is basically random, and anything seems better than random to me.
But anyway, I think that this should not block merging the PR